_posts/2022-07-14-trino-updates-to-java-17.md

```diff
@@ -0,0 +1,24 @@
+---
+layout: post
+title:  "Trino Updates to Java 17"
+author: "Cole Bowden"
+excerpt_separator: <!--more-->
+---
+
+You've already read the title, and it's exciting news - as of Trino version 390,
+which releases today, Trino has officially been updated from Java 11 to Java 17.
+This comes with a few changes, the most important of which is that if you've
+been running Trino on Java 16 or older, you'll need to update Java to utilize
+Trino versions 390 and later. It's also worth mentioning that newer versions of
+Java, such as Java 18 or 19, are not supported - they might work, but they
+haven't been tested or benchmarked - Java 17 is the new, recommended version for
+Trino.
+
+excerpt_separator: <!--more-->
+
+The reason this change is exciting is that using a new and better version of
+Java will allow us to better improve Trino, too! With new language features,
+there are more tools in the development toolkit, and with the update leading to
+the Java language performing slightly better on the whole, you may see some
+small, across-the-board performance improvements when switching from Java 11 to
+Java 17. When you've got the time, we strongly recommend making the upgrade!
```

Comment:
add conclusion section that we are excited and that you can learn more in 


https://trino.io/episodes/36.html

and we also mentioned this in 

https://trino.io/episodes/35.html

and have planned and worked on this with link to JIRA issue

And give a shout out to @wendigo for all the hard work